uplink/types.py

```diff
@@ -289,6 +302,16 @@ def modify_request(self, request_builder, value):
 
 
 class QueryMap(TypedArgument):
+    """Sets a mapping of query arguments.
+
+    If the API you are using accepts multiple query arguments,
+    you can include them all in your function method by using the
+    format: `<query argument>:uplink.QueryMap`
+
+    Args:
+        **params: A number of query argument accepted by the API.
```

Comment:
Like you did with `HeaderMap`, I think "mapping" would be more descriptive than "number" here.